google-maps/ios/Plugin/Map.swift

```diff
@@ -99,25 +103,39 @@ public class Map {
                 "longitude": self.config.center.lng,
                 "zoom": self.config.zoom
             ]
-            if let bridge = self.delegate.bridge {
 
+            if let bridge = self.delegate.bridge {
                 for item in bridge.webView!.getAllSubViews() {
                     let isScrollView = item.isKind(of: NSClassFromString("WKChildScrollView")!) || item.isKind(of: NSClassFromString("WKScrollView")!)
-                    if isScrollView {
+                    let isBridgeScrollView = item.isEqual(bridge.webView?.scrollView)
+
+                    if isScrollView && !isBridgeScrollView {
                         (item as? UIScrollView)?.isScrollEnabled = true
 
-                        let isWidthEqual = round(Double(item.bounds.width)) == self.config.width
-                        let isHeightEqual = round(Double(item.bounds.height)) == self.config.height
+                        let height = Double((item as? UIScrollView)?.contentSize.height ?? 0)
+                        let width = Double((item as? UIScrollView)?.contentSize.width ?? 0)
+                        let actualHeight = round(height / 2)
+
+                        print("\(self.config.width) x \(self.config.height)")
```

Comment:
Are these leftovers, or things we want logged?

Reply:
They were just for debugging - our existing test app seems to be working fine with new code, and I've directed a few Discord users to test out this branch, so I think we are good to go now.